public/script.js

```diff
@@ -76,8 +76,8 @@ function renderAccount(props) {
         spiritPointsCount: res.data.spiritPointsCount,
         magicPointsCount: res.data.magicPointsCount,
       });
-      allPlayers.setState({
-        showAllPlayers: false,
+      getPlayers.setState({
+        showGetPlayers: false,
```

Comment:
variable naming: a name that starts with "show" sounds more like a function than a boolean variable. Something like "shouldShow..." might be better for a boolean.

Reply:
I agree. Will change the variable name.